packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -21,11 +21,11 @@ export const richTextEditorStyle = (props: { minHeight: string; maxHeight: strin
 /**
  * @private
  */
-export const richTextEditorWrapperStyle = (theme: Theme, addTopOffset: boolean, addRightOffset: boolean): string => {
+export const richTextEditorWrapperStyle = (theme: Theme, addTopOffset: boolean): string => {
```

Comment:
I'm not sure what the best practice is on web, but IMO instead of having a `boolean` to just add extra padding to the top, I feel like we should allow input of a Rect(top, right, bottom, left) and provide a bit more flexibility going forward.

Reply:
For most the styles on web we seem to use boolean to indicate a difference in the style.

<img width="919" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/2b632306-bbb5-471f-8fa9-60582f08f867">

<img width="703" alt="image" src="https://github.com/Azure/communication-ui-library/assets/73612854/acaaed9c-8bd7-42e8-b21c-37e27b0f56fd">

So i think maybe keeping the boolean is the right choice here but changing the name to be something more descriptive. 

In this case naming it: `isFormattingShown`?

Reply:
Chatted offline. Seems consistent with everything else, if we want to in the future change this concept we can do it as an overarching change for all styles.